src/components/customized-components/ClinicCard/ClinicCard.component.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Button } from "antd";
+import { EnvironmentFilled, StarFilled } from "@ant-design/icons";
+import "./ClinicCard.style.scss";
+
+const ClinicCardComponent = (props) => {
```

Comment:
change this into {imgSrc, name, serviceArray,...} or pass an object into this, props is confusing